pkgs/development/python-modules/hcipy/default.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  setuptools-scm,
+
+  # dependencies
+  numpy,
+  scipy,
+  matplotlib,
+  pillow,
+  pyyaml,
+  astropy,
+  imageio,
+  xxhash,
+  numexpr,
+  asdf,
+
+  # tests
+  pytest,
+  pytest-cov,
+  pytestCheckHook,
+  mpmath,
+  dill,
+  flake8,
+}:
+
+buildPythonPackage rec {
+  pname = "hcipy";
+  version = "0.6.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Kt2T6EgCrPTJGiygjmFUh1HH1DthFcZ2mMlnlu0HxsE=";
+  };
+
+  build-system = [ setuptools-scm ];
+
+  dependencies = [
+    numpy
+    scipy
+    matplotlib
+    pillow
+    pyyaml
+    astropy
+    imageio
+    xxhash
+    numexpr
+    asdf
+  ];
+
+  pythonImportsCheck = [ "hcipy" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest
+    pytest-cov
```

```suggestion
    pytest-cov-stub
```
We don't need coverage information.

---

I have adjusted the dependency.